.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,17 @@
+name: Deploy docs
+
+on:
+  # Runs on pushes targeting the `main` branch. Change this to `master` if you're
+  # using the `master` branch as the default branch.
```

I would remove this comment because the `master` branch is no longer the default for GitHub based projects.

---

habe ich nicht rein. kommt von vitepress. ja stimmt das mit master ist nur für alte proejktet ich tue es gerne raus
```suggestion
  # Runs on pushes targeting the `main` branch. 
```